chef.py

```diff
@@ -178,8 +179,9 @@ def fetch_video(video):
 
     # Add subtitles in whichever languages are available.
     for language in subtitle_languages:
-        video_node.add_file(LanguagePatchedYouTubeSubtitleFile(
-            youtube_id=youtube_id, youtube_language=language))
+        if getlang_patched(language):
```

Comment:
@jtamiace This is the key additional line needed to get this to work for those YouTube subtitle languages that don't have a corresponding language code in le-utils ... sorry about all the trouble, will look into making this smoother for future chef runs!

(Then all the changes to requirements.txt can be reverted too.)